lambda/handler.py

```
@@ -0,0 +1,71 @@
+    if os.path.exists(path):
+        os.chmod(path, 0o755)
+
+
+def _run_scripts(workdir: str) -> dict:
+    install_path = os.path.join(workdir, "install_ansible_modules.sh")
+    playbook_path = os.path.join(workdir, "playbook_command.sh")
+
+    _chmod_if_exists(install_path)
+    _chmod_if_exists(playbook_path)
+
+    install = subprocess.run(["bash", "-lc", "./install_ansible_modules.sh"], cwd=workdir, capture_output=True, text=True)
```

Ruff (0.12.2)

Starting a process with a partial executable path

(S607)